bloom-housing/bloom
# 2098/fix partner applications grid display

## Issue Overview

This PR addresses [#issue](https://github.com/bloom-housing/bloom/issues/2098)


## Description
When an application had several preferred units selected it was messing up the partner's listing -> applications table

## How Can This Be Tested/Reviewed?
Create or edit an application on a listing
on that application, add several preferred units 
now go to the partner's portal, and go the table of applications for the listing you created (something like https://dev-partners-bloom.netlify.app/listings/5fc338b3-45bd-4054-a383-869f2fe02ec4/applications )

there should now be the correct number displayed on that table, depending on the number that should show

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
